pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -271,7 +272,16 @@ func (s *Service) DeleteLoadBalancer(openStackCluster *infrav1.OpenStackCluster,
 	}
 
 	record.Eventf(openStackCluster, "SuccessfulDeleteLoadBalancer", "Deleted load balancer %s with id %s", lb.Name, lb.ID)
-	return nil
+
```

Comment:
/cc @seanschneeweiss @chrischdi can you please review this part if it breaks something for us

Reply:
Let's resolve this for now

Reply:
However: I think this is ok :+1:

Reply:
Thx! :)